### PR TITLE
Embedding text messages in an Image and extracting it from the image at the receivers end.

### PR DESCRIPTION
**Pull request by WinterSoldier13**

### CODE REVIEW

File: Encrypt_decrypt_text_in_images/Decrypter.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import cv2
+import numpy as np
+import time
+
```
Comment (HarshCasper) [inline Encrypt_decrypt_text_in_images/Decrypter.py:4]: Please remove unnecessary Line Breaks

File: Encrypt_decrypt_text_in_images/Decrypter.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+arr1 =[]
+c=0
+
+def two_dim_conv(x):
+    c=0
+    for i in x:
+        for j in i:
+            for k in j:
+
+
```
Comment (HarshCasper) [inline Encrypt_decrypt_text_in_images/Decrypter.py:22]: Remove unnecessary line breaks

File: Encrypt_decrypt_text_in_images/Decrypter.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+l = arr[0]
+string_arr = arr[1:l+1]
+
+for i in string_arr:
+    str += chr(i)
+
+
+
+print(str)
+print("Intercercepted message length >>> ",end=' ')
```
Comment (HarshCasper) [inline Encrypt_decrypt_text_in_images/Decrypter.py:40]: Make use of proper Console Messages

File: Encrypt_decrypt_text_in_images/Encrypter.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+b, g, r = svimg.split()
+svimg = Image.merge("RGB", (r, g, b))
+
+svimg.save('output.png')
+print("DONE!!!")
+print("Total time taken: ",end=' ')
+print(time.time()-now)
+
+#WinterSoldier13
```
Comment (HarshCasper) [inline Encrypt_decrypt_text_in_images/Encrypter.py:61]: You can add your Name up above as `___author___`

File: Encrypt_decrypt_text_in_images/Readme.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<h1>How to use</h1>
+<br>
```
Comment (HarshCasper) [inline Encrypt_decrypt_text_in_images/Readme.md:2]: Please try to make the `README` more descriptive.

File: Encrypt_decrypt_text_in_images/Decrypter.py (hunk 1 above)
Comment (WinterSoldier13) [reply Encrypt_decrypt_text_in_images/Decrypter.py:4]: Done

File: Encrypt_decrypt_text_in_images/Decrypter.py (hunk 2 above)
Comment (WinterSoldier13) [reply Encrypt_decrypt_text_in_images/Decrypter.py]: Done

File: Encrypt_decrypt_text_in_images/Encrypter.py (hunk 4 above)
Comment (WinterSoldier13) [reply Encrypt_decrypt_text_in_images/Encrypter.py]: Done

File: Encrypt_decrypt_text_in_images/Readme.md (hunk 5 above)
Comment (WinterSoldier13) [reply Encrypt_decrypt_text_in_images/Readme.md:2]: Done

**Pull request closed without merging**